src/hooks/useKeystore.ts

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { useStore } from "./useStore";
+import { useRLN } from "./useRLN";
+import { SEPOLIA_CONTRACT } from "@waku/rln";
```

Comment:
We can but them image most likely will be ~1Gb.
Right now image has `http` server and `html+css`, if we can to pass env variables then JS should be built and that requires `node.js` image which is not small

@alrevuelta how important it is to keep image small for your case?

Reply:
Isn't there a way to improve this? Its a very simple webapp, 1Gb seems too much. What is taking that huge amount of space?

For reference, nwaku image is 50Mb.

> how important it is to keep image small for your case?

Its indeed important because pulling an image of 1Gb takes some time to download.

Reply:
I understand your frustration, but, it's not the app that is `1Gb` - it is `node.js`'s docker image that it taking that much